openfl-workspace/torch_cnn_histology/plan/plan.yaml

```diff
@@ -36,3 +36,6 @@ tasks:
 
 assigner:
   defaults: plan/defaults/assigner.yaml
+
+compression_pipeline :
```

Comment:
If there's an existing plan.yaml will it fail given that it doesn't have the "compression_pipeline" field?

Reply:
No, it will default to `NoCompressionPipeline`